tier2-icons.html

```diff
@@ -108,18 +108,18 @@
           <path fill-rule="evenodd" d="M24 4v2a1 1 0 0 1-2 0V4a2.006 2.006 0 0 0-2-2h-2a1 1 0 0 1 0-2h2a4 4 0 0 1 4 4zm0 14v2a4 4 0 0 1-4 4h-2a1 1 0 0 1 0-2h2a2.006 2.006 0 0 0 2-2v-2a1 1 0 0 1 2 0zM7 1a1 1 0 0 1-1 1H4a2.006 2.006 0 0 0-2 2v2a1 1 0 0 1-2 0V4a4 4 0 0 1 4-4h2a1 1 0 0 1 1 1zm0 22a1 1 0 0 1-1 1H4a4 4 0 0 1-4-4v-2a1 1 0 0 1 2 0v2a2.006 2.006 0 0 0 2 2h2a1 1 0 0 1 1 1zm3-15.5l6 4.8-6 4.2v-9m0-2a2 2 0 0 0-2 2v9a2 2 0 0 0 3.147 1.638l6-4.2a2 2 0 0 0 .1-3.2l-6-4.8A2 2 0 0 0 10 5.5z"/>
         </g>
         <g id="change-file">
-          <g fill-rule="evenodd">
+          <g fill="#73787B" fill-rule="evenodd">
```

Comment:
I strongly suspect this, `navigate`, and `self-assessment` are supposed to be #72777A, but I will verify with Sarah.